runtime-modules/content/src/lib.rs

```diff
@@ -235,7 +235,11 @@ decl_storage! {
 
         /// Global weekly NFT limit.
         pub GlobalWeeklyNftLimit get(fn global_weekly_nft_limit):
-            LimitPerPeriod<T::BlockNumber>;
+        LimitPerPeriod<T::BlockNumber>;
+
+        /// NFT limits enabled or not
+        /// Can be updated in flight by Lead
```

Comment:
```suggestion
        /// Can be updated by the Council
```

Reply:
I added `by the Council` but only `root` can toggle the variable at the moment